sage-android/Sage/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="red500">#F44336</color>
```

Comment:
Take a look at the Design specs in the Shared SAGE folder for the colors that iOS is using. I guess as long as they are close enough, it's fine, but I'd like to keep consistent colors across iOS and Android